### ISSUE TITLE
POST /{db}/_session missing from SG 1.2 API Docs

### ISSUE THREAD

**Issue by ajres** [couchbase/sync_gateway#1633]
The SG [1.1 docs](http://developer.couchbase.com/documentation/mobile/1.1.0/develop/references/sync-gateway/rest-api/authentication/post-session/index.html) include 'POST /{db}/_session' 

The [1.2 docs](http://developer.couchbase.com/documentation/mobile/1.2/develop/references/sync-gateway/rest-api/authentication/index.html) do not.

Not sure if this has been fixed and awaiting publishing.

**Comment by adamcfraser** [comment]
It's there under the admin API - looks like it's been removed from the public API (inadvertently I assume).

**Comment by tleyden** [comment]
Looks like this has been deployed to http://developer.couchbase.com/documentation/mobile/1.2/develop/references/sync-gateway/admin-rest-api/session-admin/index.html

```
IN THIS GUIDE

GET /{db}/_session/{session-id}
POST /{db}/_session
DELETE /{db}/_session/{session-id}
DELETE /{db}/_user/{name}/ _session/{session-id}
DELETE /{db}/_user/{name}/_session
Admin REST APIs for sessions enable administration of authentication sessions. All of the resource URIs contain a path segment for a specific database. The following table lists the HTTP methods and URI resource patterns for the Admin REST APIs for sessions:

HTTP method URI pattern of the resource Description
GET /{db}/_session/{session-id} Gets information about a session
POST    /{db}/_session  Creates a session
DELETE  /{db}/_session/{session-id} Deletes a session
DELETE  /{db}/_user/{name}/_session/{session-id}    Deletes a session for the specified user
DELETE  /{db}/_user/{name}/_session Deletes all sessions for the specified user
```

**Comment by adamcfraser** [comment]
@tleyden That's admin API - the issue was that it's missing in the public API.

**Comment by tleyden** [comment]
Oops, thanks for catching that!

**Comment by jamesnocentini** [comment]
Fixed in swagger spec and 1.3 docs.
